fable/include/fable/conf.hpp

```diff
@@ -118,25 +122,25 @@ class Conf {
   /**
    * Return the root of the current JSON as a JSON pointer.
    */
-  std::string root() const { return (root_.empty() ? "/" : root_); }
+  [[nodiscard]] std::string root() const { return (root_.empty() ? "/" : root_); }
```

Comment:
I think it's because I'm only using it for error messages. Could look into it and document accordingly.

- [ ] Look into why JsonPointer not directly used.